core/src/autogluon/core/searcher/bayesopt/gpautograd/gp_regression.py

```diff
@@ -132,25 +123,22 @@ def fit(self, X, Y, profiler: GPMXNetSimpleProfiler = None):
         fail, log messages are written. If all restarts fail, the current
         parameters are not changed.
 
-        :param X: data matrix X of size (n, d) (type mx.nd)
-        :param Y: matrix of targets of size (n, m) (type mx.nd)
+        :param X: data matrix X of size (n, d)
+        :param Y: matrix of targets of size (n, m)
```

Comment:
documentation may want to clarify why Y can be m-dimensional target

Reply:
Fixed docstring